test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/client/util/Kjar.java

```diff
@@ -28,28 +31,38 @@
     STATELESS_SESSION("org.kie.server.testing", "stateless-session", "1.0.0.Final");
 
     private String groupId;
-    private String name;
+    private String artifactName;
     private String version;
+    private String projectName;
 
-    private Kjar(String groupId, String name, String version) {
+    private Kjar(String groupId, String artifactName, String version) {
+        this(groupId, artifactName, version, artifactName);
+    }
+
+    private Kjar(String groupId, String artifactName, String version, String projectName) {
         this.groupId = groupId;
-        this.name = name;
+        this.artifactName = artifactName;
         this.version = version;
+        this.projectName = projectName;
     }
 
     public String getGroupId() {
         return groupId;
     }
 
     public String getName() {
-        return name;
+        return artifactName;
```

Comment:
Also it would be good to change of the getter method.

Reply:
I changed the method's name